tests/Console/ConfigurationResolverTest.php

```diff
@@ -1060,7 +1061,7 @@ public function testResolveCommandLineInputOverridesDefault()
         $this->assertSame(['php_unit_construct' => true], $resolver->getRules());
         $this->assertFalse($resolver->getUsingCache());
         $this->assertNull($resolver->getCacheFile());
-        $this->assertInstanceOf(\PhpCsFixer\Differ\SebastianBergmannDiffer::class, $resolver->getDiffer());
+        $this->assertInstanceOf(\PhpCsFixer\Differ\UnifiedDiffer::class, $resolver->getDiffer());
```

Comment:
Please add a test that covers various combinations of `diff` and `diff-format` options.